packages/styles/components/_c.stepper.scss

```diff
@@ -57,36 +57,36 @@
     &--validated {
       #{$parent} {
         &__indicator {
-          border-color: $color-primary-01-400;
+          border-color: $color-stepper-item-current-background;
         }
 
         &__detail {
-          @include set-font-weight('semi-bold');
+          @include set-font-weight("semi-bold");
         }
       }
     }
 
     &--current {
       &::before {
-        background-color: $color-primary-01-400;
+        background-color: $color-stepper-item-current-background;
       }
 
       #{$parent} {
         &__indicator {
-          background-color: $color-primary-01-400;
-          color: $color-grey-000;
+          background-color: $color-stepper-item-current-background;
+          color: $color-font-lightest;
```

Comment:
Replace -lightest to -invert

Reply:
@caqueste as seen together, this point will be fixed in a new PR on the `@next` version